com.unity.render-pipelines.core/Runtime/Common/DynamicResolutionHandler.cs

```diff
@@ -78,6 +102,17 @@ private void Reset()
         /// </summary>
         public Vector2Int finalViewport { get; set; }
 
+        /// <summary>
+        /// By default, dynamic resolution scaling is turned off automatically when the source matches the final viewport (100% scale).
+        /// That is, DynamicResolutionEnabled and SoftwareDynamicResIsEnabled will return false if the scale is 100%.
+        /// For certain upscalers, we dont want this behavior since they could possibly include anti aliasing and other quality improving post processes.
+        /// Setting this to true will eliminate this behavior.
+        /// </summary>
+        public bool allowDynamicResolutionOnAllPercentages
```

Comment:
I feel the name might be confusing, I'd change it to something like "Run Upscaler filter on full resolution" (not exactly this, but in this vein)

Reply:
Changed this to runUpscalerFilterOnFullResolution